### PR TITLE
164462685 Implement application navbar

### PR DESCRIPTION
**Pull request by codjoero**
#### Description
#
The App currently has no navigation bar, this adds it with responsiveness and response to the state change for when either a user is logged in or not.

#### Type of change
#
- New feature

#### How Has This Been Tested?
#
- Unit tests testing individual functionalities

#### Screenshots:
#
Screenshot 2019-03-14 at 16 54 35
Screenshot 2019-03-15 at 07 18 20

#### Relevant **Pivotal Tracker** stories
#
- [164462685](https://www.pivotaltracker.com/story/show/164462685)

### CODE REVIEW

**Review by missvicki — APPROVED** [review]
I like the fact that you catered for responsiveness in a very smart way. It looks good to me.

**Comment by oma0256** [comment]
Great work @codjoero with the navbar but I think it would have been better if the logo was clickable and it would redirect to the home page otherwise good work

**Comment by pbkabali** [comment]
This is nice work @FahdJamy and @codjoero. The sign-in link/button seems to disappear in mobile view thou. Could you please check on that?

**Review by karuhanga — APPROVED** [review]
LGTM

**Review by pbkabali — APPROVED** [review]

File: src/App.test.jsx [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
 
 
-const middleware = [thunk];
-const mockStore = configureStore(middleware);
+const mockStore = createStore(
+    reducer, // , pass in the main reducer
+    applyMiddleware(thunk) // , apply any middleware we are making use of
+);
+
+// const props = {
```
Comment (collinewait) [inline src/App.test.jsx]: Remove commented code

File: src/components/tests/NavBar.test.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Navbar } from '../Navbar';
+import navbarState from './fixtures/navBarState';
+// import branding from '../../assets/images/branding.png';
+
```
Comment (collinewait) [inline src/components/tests/NavBar.test.jsx]: Remove commented code

Comment (codjoero) [reply src/components/tests/NavBar.test.jsx]: Nice catch, thanks. Its done

**Pull request merged**